smart_proxy.go

```diff
@@ -128,7 +129,21 @@ func startServer() ExitCode {
 		log.Info().Msg("AMSClient successfully created")
 	}
 
-	serverInstance = server.New(serverCfg, servicesCfg, amsClient, groupsChannel, errorFoundChannel, errorChannel)
+	redisClient, err := services.NewRedisClient(redisConf)
```

Comment:
is it possible that the client gets outdated somehow and we need to re-initialize it? not sure tbh, I'm probably over-worrying

Reply:
I don't think so, the connection is handled wholly by the go-redis pkg

Reply:
discussed on slack, let's trust :)